prowler/providers/gcp/lib/service/service.py

```diff
@@ -66,3 +67,16 @@ def __is_api_active__(self, audited_project_ids):
                     f"{error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
                 )
         return project_ids
+
+
+def generate_client(
```

Comment:
I think this should be moved to the class, right?

Reply:
Moved during merge